runtime/parser/init.go

```diff
@@ -46,6 +46,28 @@ mock_users:
 		return err
 	}
 
+	// Determine the driver based on olap
+	var driver string
+	switch olap {
+	case "", "duckdb":
+		driver = "duckdb"
+	case "clickhouse":
+		driver = "clickhouse"
+	default:
+		// Default to duckdb for unknown olap
+		driver = "duckdb"
+	}
+
+	// Create the connector YAML
+	connectorYAML := fmt.Sprintf(`type: connector
+driver: %s
+`, driver)
```

Comment:
Can you customize this for each connector? E.g. for Clickhouse, let's add `managed: true`. In the future, we may also want to add some commented out options or relevant docs links